medusa/helpers/externals.py

```diff
@@ -41,8 +27,9 @@ def trakt_request(api, url):
                 app.TRAKT_ACCESS_TOKEN = api.access_token
                 app.TRAKT_REFRESH_TOKEN = api.refresh_token
                 app.instance.save_config()
-        except (TokenExpiredException, TraktException) as e:
-            logger.info(u'Could not use Trakt to enrich with externals. Cause: {cause}', cause=e)
+        except (TokenExpiredException, TraktException) as error:
+            log.info(u'Could not use Trakt to enrich with externals: {0}',
+                     error)
```

Comment:
we should log only the e.message and not entire error object
At least its the "default" for most of all code